drivers/SmartThings/matter-appliance/src/matter-dishwasher/init.lua

```diff
@@ -125,27 +125,27 @@ local function dishwasher_alarm_attr_handler(driver, device, ib, response)
   local isWaterFlowVolumeAlarm = false
 
   local state = ib.data.value
-  if state & clusters.DishwasherAlarm.types.AlarmMap.INFLOW_ERROR > 0 then
+  if state & clusters.DishwasherAlarm.types.AlarmBitmap.INFLOW_ERROR > 0 then
```

Comment:
Thank you for pointing this out @SathvikGatti ! I think we have two options then for moving forward:
1) Add the alias support to the embedded cluster definitions. This way, we can add an alias that maps `AlarmMap -> Alarm Bitmap` so that the name `AlarmMap` can still be used in the driver when the lua libs api is <= 10. This would require adding the `alias` files to the driver embedded cluster definitions.
2) We could keep the name updates in the embedded cluster definition to match the updates made in Matter 1.3, and then we change the `require` statement to from `if version.api < 10 then` to `if version.api < 11 then`. This means that we will need to use the embedded cluster definition for longer (i.e. until all platforms are updated to use the Matter 1.3 lua libs), but I think it is a cleaner solution overall. Keeping the embedded cluster definitions means the driver package will be larger from a memory perspective, but the driver memory when running will actually be either the same or less than when using  the lua libs definitions. Since we are not close to running out of space in the driver package yet, I think option 2 could be the preferred option.

I think that option 2 is probably the easiest, it will just mean that we will have to keep all embedded clusters up to date with each matter specification, and then updating the drivers to use the latest name for embedded cluster values. @HunsupJung do you have a preference between these options? If not, I think we should go with option 2.

Reply:
Actually, on further thought, I think we should actually not update the embedded clusters at all. I think they should remain as they are with the name changes for Matter 1.3. Basically, this is if we continue to use the original name of the attributes, it will allow the same driver to work across all different lua libs versions and avoid the issue that @SathvikGatti originally mentioned. I have a little bit of background on this here: https://smartthings.atlassian.net/wiki/spaces/~62de3b63b6b0b70770d7905d/pages/3406629893/Handling+Cluster+Changes+Across+Matter+Specs+and+FW+Versions

So, I think I will revert [5c0495a](https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/pull/1563/commits/5c0495ad5dc9a519cc89824cc8d752c349e3dbcf) and we can leave the embedded cluster definitions as they are.